crawl-ref/source/dat/strings/de/mutations.txt

```diff
@@ -1185,13 +1185,13 @@ You have a snake-like lower body.
 Ihr habt einen schlangenartigen Unterkörper.
 %%%%
 You can use your snake-like lower body to constrict enemies.
-Ihr könnt Euren schlangenartigen Unterkörper einsetzen, um Feinde zu umschlingen.
+Ihr könnt Euren schlangenartigen Unterkörper einsetzen, Eure Feinde zu umschlingen.
 %%%%
 Your lower body turns into a snake tail.
 Euer Unterkörper verwandelt Euch in einen Schlangenschwanz.
 %%%%
 Your tail grows strong enough to constrict your enemies.
-Euer Schwanz wird stark genug, um Eure Feinde zu umschlingen.
+Euer Schwanz wird stark genug, Eure Feinde zu umschlingen.
```

Comment:
I'd phrase it this way:
Euer Schwanz wächst und ist nun stark genug, Eure Feinde zu umschlingen.

Reply:
done